lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -160,6 +160,25 @@ static const std::vector<std::string> kSchemaNiceNames = {
 // clang-format on
 #endif
 
+//! \brief Change the cursor to wait state when start() is called and restore it on destruction.
+struct MWaitCursor
+{
+    bool _started { false };
+
+    MWaitCursor() = default;
+    void start()
```

Comment:
Why do we need the start() method?  Can't we just start in the constructor, and get rid of the _started data member?  Also not sure about the "M" prefix in the name.

Reply:
This is how this class should look like if it was part of OpenMaya, and hopefully one day.